viz/cmd/install.go

```diff
@@ -36,6 +36,7 @@ var (
 		"templates/tap-injector-rbac.yaml",
 		"templates/tap-injector.yaml",
 		"templates/web.yaml",
+		"templates/service-profiles.yaml",
```

Comment:
Super specific nit: This is unrelated to your change but the slice variable that holds this string is named `templatesVIz`, I think it should be `templateViz` instead.

Reply:
Done!